### ISSUE TITLE
Weird Zet Aspects and SOTS Blessing intereaction

### ISSUE THREAD

**Issue by ConqAra** [William758/ZetAspects#81]
ok so i opened a blessing and selected an item before escaping out of it cuz two fire beetles were attacking me and both straight up just dropped their aspect after I killed them. I have default config for ZetAspects and use the ESCapism mod if that helps. There was no messages that they dropped either, just really odd behavior.
![image](https://github.com/user-attachments/assets/bbdc2b19-381e-4807-ac51-9f2ab32362d8)

Log: 
[LogOutput.log](https://github.com/user-attachments/files/17103969/LogOutput.log)

**Comment by William758** [comment]
Yeah that's something. I'd say I'm fairly confident that ZetAspects didn't drop them. The order of events on a successful drop is increase the run drop counter (the logs don't show this happening), create the chat message (also didn't happen), then finally create the droplet. I ignore equipment drop chance OnCharacterDeath and that change doesn't show any errors either. Maybe another mod made changes that override my changes and you got EXTREMELY lucky or some other mod effect has a chance to drop monster equipment. I don't have the dlc to test the shrine itself but yeah that's just really weird.

**Comment by ConqAra** [comment]
Cant reproduce issue
